### ISSUE TITLE
FEATURE: Jump to Note

### ISSUE THREAD

**Issue by farling42** [MiahNelah/better-entity-link#11]
Foundry provides a "Jump to Pin" option in the Journal Sidebar.

It would be nice to have this available on links to journals.

Foundry's code for sidebar button is:

```
return options.concat([
      {
        name: "SIDEBAR.JumpPin",
        icon: '<i class="fas fa-crosshairs"></i>',
        condition: li => {
          const entry = game.journal.get(li.data("document-id"));
          return !!entry.sceneNote;
        },
        callback: li => {
          const entry = game.journal.get(li.data("document-id"));
          return entry.panToNote();
        }
      }
    ]);
```

**Comment by KelSolaar** [comment]
Ah great, was coming for that one! Guess you saw the discussion on Discord? :)

**Comment by MiahNelah** [comment]
Thanks for your feedback AND code 👍 I released v1.2.4 with this feature. Enjoy !
